doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-rule-table-entries-ref.adoc

```diff
@@ -50,7 +50,8 @@ To include a pattern without constraints, you can write the pattern in front of
 +
 If the pattern ends with `eval`, code snippets produce boolean expressions for inclusion into a pair of parentheses after `eval`.
 +
-The pattern can terminate with the `@watch` annotation, used to customize the properties on which the pattern is reactive.
+You can terminate the pattern with `@watch` annotation, which is used to customize the properties on which the pattern is reactive. 
```

Comment:
How about this minor rewrite that removes on instance of "which"?
You can terminate the pattern with the @watch annotation to customize the properties on which the pattern is reactive.

Reply:
Done, thanks Michele